### ISSUE TITLE
desperately trying to style a pricing (or any) table in unyson

### ISSUE THREAD

**Issue by thebais** [ThemeFuse/Unyson#2943]
Hi,
Ive tried everything imaginable to try and style the tables within unison framework, can anyone point me in the right direction? I managed to change the target class within the child framework, but even though I can see clearly from the source that the table class has changed, I cannot target it via css? Can anyone help?
Regards
Bob

**Comment by thebais** [comment]
#update 
I actually just realised I cannot apply any css, at least not by the method I've used before which is in the child template folder create a css referring to parent folder and target whatever i need in here, but even that doesn't work! I can write in the little custom css box at the backend, but the manual states only use this if its one or two small bits of css and if not use a stylesheet. Although I don't seem to be able to use a stylesheet.. What am I missing?

**Comment by ViorelEremia** [comment]
I did not understand much that you can't, but you can overwritting shortcode table see here how: http://manual.unyson.io/en/latest/extension/shortcodes/index.html?highlight=shortcode#overwriting-shortcodes
You can rewrite views, style.css etc, or it is not enough.

**Comment by thebais** [comment]
Hi,
Apologies I haven't replied to this until now.. Ive been on a major project in work and haven't been able to get back to this until now.. I made the mistake of telling a friend I would do this for them and its the sorriest thing Ive ever done. I chose a framework in the hope that they would be able to add this and that themselves therefor not having to come to me for everything, but these frameworks are sooo difficult to customise unless you're an expert at how they are structured.
I had already came across that page in my searching, but its not great for giving a real example of actually changing something in particular, so I was finding it difficult. However, for all I know I actually may have had it working.. because something funny is going on with it.. Ill explain, I was trying all these things that I seen at websites and nothing seemed to be making a difference and so out of frustration I tried writing some really simple stuff in the childs stylesheet..and I found that even changing the simple things like body and a elements weren't changing anything..
Now I come back to it tonight and they have changed everything, but worse still is the fact that I cannot remove these stupid changes and if I inspect the element and look at resources its showing my stupid changes on the stylesheet despite the fact that Ive completely removed them! How is this even possible? Is there some sort of pre-building of the css then storing it in the MYSQL DB? Its really incredible frustrating and I never had this much trouble on a wordpress website without frameworks despite admitting Im no wordpress expert, it was just easier.. unfortunately Im now committed to this one..

**Comment by thebais** [comment]
OMG MFG!!!! I spent hours and hours and hours playing with it that night I originally wrote this, and the night before that..then another hour or so tonight before I realised that changes where not being reflected because of caching.. This should have a bloody health warning!  All I had to do was make my changes and in the developers part hit shift-command-R and it is now showing all the changes I made. I have to say in all my years of programming.. and in numerous languages including some that have went the way of the dinosaurs like RPG because they supposedly weren't good enough,  I have never been so frustrated by something so idiotic! Anyway problem solved finally..
